src/node/NodeOptions.hs

```diff
@@ -222,6 +207,25 @@ addrNodeOption =
         metavar "HOST:PORT" <>
         help "Identifier of a node in a Kademlia network"
 
+tlsParamsOption :: Parser TlsParams
```

Comment:
@Martoon-00 how is light wallet working ? Do we have some http endpoints open where light wallet can do some actions without TLS on ? If that is the case I would turn TLS on there